<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageOrder</class>
 <widget class="QWidget" name="StageOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="wMenu" native="true">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#wMenu{
background-color: rgb(244, 244, 244);

}

QPushButton { 
	border:none; 
	background-color: rgba(255, 255, 255,0);
	font: 700 20px &quot;Microsoft JhengHei UI&quot;; 
    min-width: 4em; 
    min-height:1.2em; 
	padding: 6px; 
	color: gray;
border-radius:0px;
	}

QPushButton:hover 
{ 
background-color: rgb(198, 200, 223); 
border-style: outset; 
border-width: 2px; 
font: 800 22px &quot;Microsoft JhengHei UI&quot;; 
}
QPushButton:pressed { 
border-style: outset; 
border-width: 0px; 
background-color: rgb(67, 67, 67);
}
 QPushButton:checked { 
color:rgb(255,255,255);
background-color: rgb(67, 67, 67);
border-style: outset; 
border-width: 0px; 
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn0">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>價目表</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>接單儲值</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>訂單回報</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>每日報表</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>每日入帳</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>288</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="LayerCostTable" name="pageCostTable"/>
        <widget class="LayerGetOrder1" name="pageGetOrder1"/>
        <widget class="LayerGetOrder2" name="pageGetOrder2"/>
        <widget class="LayerDayReport" name="pageDayReport"/>
        <widget class="LayerDayDebit" name="pageDayDebit"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerDayReport</class>
   <extends>QWidget</extends>
   <header>LayerDayReport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerCostTable</class>
   <extends>QWidget</extends>
   <header>LayerCostTable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerGetOrder1</class>
   <extends>QWidget</extends>
   <header>LayerGetOrder1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerGetOrder2</class>
   <extends>QWidget</extends>
   <header>LayerGetOrder2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerDayDebit</class>
   <extends>QWidget</extends>
   <header>LayerDayDebit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
